src/components/study/Eligibility.vue

```diff
@@ -92,11 +92,7 @@ export default {
       this.step += 1;
     },
     doSubmit() {
-      if (this.comfort === this.$t('eligibility-screen.q3.have-android') && window.queryParams.android !== "true") {
-          window.location = ANDROID_FORM_LINK;
-          return;
-      }
-      var validResidence = (this.residence === this.$t('eligibility-screen.q2.in-US') || this.residence === this.$t('eligibility-screen.q2.in-other'));
+      var validResidence = (this.residence !== this.$t('eligibility-screen.q2.outside-eligible-area'));
```

Comment:
all other options are eligible, so verify residence is not the ineligible choice